etree.go

```diff
@@ -894,6 +921,58 @@ func (c *CharData) writeTo(w *bufio.Writer, s *WriteSettings) {
 	escapeString(w, c.Data, m)
 }
 
+// NewPlainTextData creates a parentless XML character data entity.
+func NewPlainTextData(data string) *PlainTextData {
+	return newPlainTextData(data, false, nil)
+}
+
+// newPlainTextData creates an XML character data entity and binds it to a parent
+// element. If parent is nil, the PlainTextData token remains unbound.
+func newPlainTextData(data string, whitespace bool, parent *Element) *PlainTextData {
+	c := &PlainTextData{
+		Data:       data,
+		whitespace: whitespace,
+		parent:     parent,
+	}
+	if parent != nil {
+		parent.addChild(c)
+	}
+	return c
+}
+
+// CreatePlainTextData creates an XML character data entity and adds it as a child
+// of element e.
+func (e *Element) CreatePlainTextData(data string) *PlainTextData {
+	return newPlainTextData(data, false, e)
+}
```

Comment:
Rename to `CreateTextData`.

Reply:
Added `CreateTextData` and `NewTextData`